package/CHANGELOG

```diff
@@ -48,11 +48,18 @@ Enhancements
    (Issue #4948 #2874, PR #4965)
  * Enables parallelization for analysis.lineardensity.LinearDensity
    (Issue #4678, PR #5007)
- * Remove `default` channel from RTD conda env. (Issue # 5036, PR # 5037)
+ * Enabled selection of backend for distance calculations in RDF classes
+   (PR #5038)
+ * Modified functions in lib.distances to add the backend keyword,
+   so that it gets passed through from the calling functions and classes
+   (PR #5038)
+ * Moved distopia checking function to common import location in
+   MDAnalysisTest.util (PR #5038)
 
 Changes
  * Removed undocumented and unused attribute
    `analysis.lineardensity.LinearDensity.totalmass` (PR #5007)
+ * Remove `default` channel from RTD conda env. (Issue # 5036, PR # 5037)   
```

Comment:
Is this still a thing? I dont see it in the diff

Reply:
This was me asking for a fix to CHANGELOG that was unrelated to this PR – sorry for the confusion.

Reply:
See https://github.com/MDAnalysis/mdanalysis/pull/5038#discussion_r2087783791